File: benchmarks/math/fibonacci/fibonacci.go
// Go Benchmark: Recursive Fibonacci
package main

import "fmt"

func fib(n int) int {
	if n <= 1 {
		return n
	}
	return fib(n-1) + fib(n-2)
}

func main() {
	result := fib(30)
	fmt.Printf("fib(30) = %d\n", result)
}

File: benchmarks/math/fibonacci/fibonacci.py
# Python Benchmark: Recursive Fibonacci
import sys
sys.setrecursionlimit(100000)

def fib(n):
    if n <= 1:
        return n
    return fib(n - 1) + fib(n - 2)

result = fib(30)
print(f"fib(30) = {result}")
